m4/awscli_help.m4

```diff
@@ -33,8 +33,9 @@ Optional arguments:
                           Specify type of AWS CLI installation. Options are:
                           "portable-exe", "system-sandbox" (default is
                           "system-sandbox")
-  --with-download-deps    Download all dependencies and use those when
-                          building the AWS CLI. If not specified, the
+  --with-download-deps    Download all Python dependencies and use those when
+                          building the AWS CLI. Note that this is best effort
```

Comment:
It would probably be good if we add the additional note of "nor continue to work in between versions of the AWS CLI" to better indicate that even if you get it working it may not continue to work in future versions.